.github/workflows/build-and-test.yaml

```
@@ -51,7 +51,8 @@ jobs:
         run: make lint
 
       - name: Test
-        run: make test
+          run: make test
```

Here is the reason. The yaml indentation is wrong 
```suggestion
        run: make test
```

---

thx for your help